src/pages/billing/Project.js

```diff
@@ -777,6 +777,7 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
       } else {
         setErrorMessage(await (error instanceof Response ? error.text() : error))
       }
+      setUpdatingProjectCost(false)
```

Comment:
In the case of no errors, this flag is cleared [here](https://github.com/DataBiosphere/terra-ui/blob/5e9c6b3f341b0d4cded62be2eda39df01bcc2999/src/pages/billing/Project.js#L768). The flag is consulted [here](https://github.com/DataBiosphere/terra-ui/blob/5e9c6b3f341b0d4cded62be2eda39df01bcc2999/src/pages/billing/Project.js#L720) to avoid multiple Ajax calls while an update is in progress.